src/service/cli/generate.js

```diff
@@ -32,13 +35,24 @@ const readFileContent = async (filePath) => {
   }
 };
 
-const generateOffers = (count, sentences, titles, categories) => (
+const generateComments = (count, comments) => (
   Array(count).fill({}).map(() => ({
+    id: nanoid(MAX_ID_LENGTH),
+    text: shuffle(comments)
+      .slice(0, getRandomInt(1, 3))
+      .join(` `),
+  }))
+);
+
+const generateOffers = (count, sentences, titles, categories, comments) => (
+  Array(count).fill({}).map(() => ({
```

Comment:
```suggestion
  new Array(count).fill({}).map(() => ({
```
все конструкторы должны вызываться с `new`